feature-push-notifications/src/main/java/io/novafoundation/nova/feature_push_notifications/data/data/subscription/RealPushSubscriptionService.kt

```diff
@@ -167,4 +170,19 @@ class RealPushSubscriptionService(
     private fun Map<String, Any>.nullIfEmpty(): Map<String, Any>? {
         return if (isEmpty()) null else this
     }
+
+    private fun <T> Map<ChainId, T>.transfromChainIdsTo16Hex(): Map<String, T> {
+        return mapKeys { (chainId, _) -> chainId.to16Hex() }
+    }
+
+    private fun List<ChainId>.transfromChainIdsTo16Hex(): List<String> {
+        return map { chainId -> chainId.to16Hex() }
+    }
+
+    private fun ChainId.to16Hex(): String {
```

Comment:
I suggest to use `hexPrefix16()`. When first reading `to16Hex` is some kind of base but not the prefix length